style.css

```diff
@@ -0,0 +1,137 @@
+body {
+	width: 98%;
+	min-width: 500px;
+	max-width: 1000px;
+	font-family: arial;
+	font-weight: bold;
+	margin: 0px auto; 
+}
+
+#container {
+	width: 100%;
+	background-color: #FF5733;
+	color: rgb(51, 255, 255);
+}
+
+#header {
+	width: 100%;
+	height: 
+}
+
+#header h1 {
+	font-size: 20px;
+	color: #DAF7A6;
+	padding-bottom: 10px;
+	text-align: center;
+	letter-spacing: 2px;
+	text-shadow: 0px 1px 0px #fff; 
+}
+
+.navbar {
+	text-align: center;
+	background-color: green;
+}
+
+.navbar ul {
+	list-style-type: none;
+}
+
+.navbar li {
+	display: inline-block;
+	border: 3px solid lightgreen;
+	border-radius: 3px 
+	padding: 10px;
+	border-radius: 5px;
+}
+
+.navbar li a {
+	text-decoration: none;
+	color: #101040;
+	font-size: 20px;
+}
+
+.navbar a:hover {
+	color: #FFC300; 
+}
+
+.main #img1 {
+	width: 200px; 
+	height: 200px;
+	display: inline-block;
+	border-radius: 200px;
+	border-color: 5px solid black;
+	margin: 10px 5px 10px 5px;
+	padding: 10px 5px 10px 5px;
+}
+
+.main #img2 {
```

Comment:
You can replace `.main #img2` with styling every image in `main` by

```
.main img {
    width: 200px;
    height: 200px;
    display: inline-block;
    border-radius: 200px;
    border-color: 5px solid black;
    margin: 10px 5px 10px 5px;
    /* padding: 10px 5px 10px 5px; */
}
```

Or did you have idea to have different styles for images?